### PR TITLE
VPS-167/authoring-tool-z-axis-control

### PR DESCRIPTION
**Pull request by itsatulbox**
## Describe the issue

Elements (textboxes, images, etc) added to the canvas could not be moved in the z axis relative to the other elements. Sidebar has some visual bugs that can be improved

## Describe the solution

Implemented a `zPosition` property for individual elements and used CSS styling to adjust its positioning along the z-axis. Fixed visual bugs in the sidebar.

## Risk

Visuals might not satisfy client

## Definition of Done

- [ ] Code peer-reviewed
- [ ] Unit tests written and passing
- [ ] Continuous Integration build passing
- [ ] Acceptance criteria met

## Reviewed By

Who reviewed your PR - for commit history once merged

### CODE REVIEW

File: frontend/src/containers/pages/AuthoringTool/CanvasSideBar/ComponentProperties/ButtonPropertiesComponent.jsx [hunk 1]
```diff
@@ -71,7 +71,7 @@ export default function ButtonPropertiesComponent({
         </Select>
       </FormControl>
       <FormControl fullWidth className={styles.componentProperty}>
-        <CustomInputLabel shrink>Next Scene</CustomInputLabel>
+        <CustomInputLabel shrink>Linked Scene</CustomInputLabel>
```
Comment (wjin-lee) [inline frontend/src/containers/pages/AuthoringTool/CanvasSideBar/ComponentProperties/ButtonPropertiesComponent.jsx:74]: I'm not sure about this one - might be missing something though. What was the reasoning behind the change?

**Review by wjin-lee — APPROVED** [review]
Overall great job!
I think the implementation could have been cleaner if it was a part of a larger refactoring of all shared properties for each canvas element, but that is a story for a later sprint :)

One comment is just a nit - feel free to bring it up during sprint planning.

**Review by wjin-lee — CHANGES_REQUESTED** [review]
Also, pls fix the frontend unit tests failing before merging.

File: frontend/src/containers/pages/AuthoringTool/CanvasSideBar/ComponentProperties/ButtonPropertiesComponent.jsx (hunk 1 above)
Comment (itsatulbox) [reply frontend/src/containers/pages/AuthoringTool/CanvasSideBar/ComponentProperties/ButtonPropertiesComponent.jsx:74]: RESOLVED: The button could take us to the previous scene/any scene not necessarily the next scene in the sequence.

**Review by wjin-lee — APPROVED** [review]
lgtm!

**Pull request merged**